GitCommands/Settings/AppSettings.cs

```diff
@@ -82,6 +85,24 @@ static AppSettings()
                 }
             });
 
+            LocalApplicationDataPath = new Lazy<string>(() =>
+            {
+                if (IsPortable())
+                {
+                    return GetGitExtensionsDirectory();
+                }
+                else
```

Comment:
this `else` is redundant and should be removed to reduce nesting

Reply:
Ok